<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encryption</class>
 <widget class="QDialog" name="Encryption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>790</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
font-size: 13pt;
color: rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
font-size: 13pt;
color: rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aes_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
font-size: 13pt;
color: rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>AES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="des_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
font-size: 13pt;
color: rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>DES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rc4_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>300</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
font-size: 13pt;
color: rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>RC4</string>
   </property>
  </widget>
  <widget class="QLabel" name="choose_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
font-size: 20pt
</string>
   </property>
   <property name="text">
    <string>Choose your picture</string>
   </property>
  </widget>
  <widget class="QLabel" name="key_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
font-size: 20pt
</string>
   </property>
   <property name="text">
    <string>Enter your key</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="key_edit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Type here!</string>
   </property>
  </widget>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15pt;
font: bold
</string>
   </property>
   <property name="text">
    <string>Your picture</string>
   </property>
  </widget>
  <widget class="QLabel" name="img_label_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>420</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15pt;
font: bold
</string>
   </property>
   <property name="text">
    <string>Encrypt Picture</string>
   </property>
  </widget>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="enimg">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>460</y>
     <width>371</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold;
font-size: 20pt;
color: rgb(85, 0, 127)</string>
   </property>
   <property name="text">
    <string>Running time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>230</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold;
font-size: 15pt</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="second_label">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>230</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold;
font-size: 15pt</string>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cbc_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
font-size: 13pt;
color: rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>CBC Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ecb_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
font-size: 13pt;
color: rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>ECB Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="team_label">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 85, 255);
font: bold</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Cryptography Theory &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Made by Team07&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
